### PR TITLE
CMRARC-834: As a CMR Dashboard User, I Would Like Review Comments to Not Extend Past the Comments Pane

### PR DESCRIPTION
**Pull request by tle1989**

### CODE REVIEW

**Comment by tle1989** [comment]
Problem: Currently, if a longer review comment is left on a record report, the comment text sometimes extends past the Review Comments pane making it difficult to read against the background image
Screenshot 2024-04-11 at 8 28 52 AM

Solution: 
Screenshot 2024-04-11 at 4 51 55 PM

File: app/views/records/show.html.erb [hunk 1]
```diff
@@ -123,7 +123,7 @@
                 <!--Column 1: the text-->
 
                 <% unless review.review_comment.blank? %>
-                  <div class="comment_text_label" style="display: flex;" id=<%="text" + getFormReviewId(review)%>><%= string_html_format(review.review_comment) %></div>
+                  <div class="comment_text_label" style="display: flex; overflow-wrap: anywhere;" id=<%="text" + getFormReviewId(review)%>><%= string_html_format(review.review_comment) %></div>
```
Comment (cgokey) [inline app/views/records/show.html.erb:126]: Suggest moving these inline styles into the comment_text_label class.

Comment (tle1989) [reply app/views/records/show.html.erb]: Done, moved these inline styles into the comment_test_label class!

**Comment by tle1989** [comment]
> In your second screenshot, it looks like the white is extending far to the right, where the first screenshot you can see the background, why is this happening?

Because the way I took the screenshot, I zoomed in more, here is the full screen 
Screenshot 2024-04-12 at 2 45 27 PM

**Review by cgokey — APPROVED** [review]

**Comment by mandyparson** [comment]
Screenshot 2024-04-12 at 2 52 46 PM
Screenshot 2024-04-12 at 2 53 46 PM

Should there be any differentiation between the two text boxes? All the comments run into each other as opposed to looking like two different entities.

**Comment by cgokey** [comment]
> Screenshot 2024-04-12 at 2 52 46 PM Screenshot 2024-04-12 at 2 53 46 PM
> Should there be any differentiation between the two text boxes? All the comments run into each other as opposed to looking like two different entities.

Agree 

> Screenshot 2024-04-12 at 2 52 46 PM Screenshot 2024-04-12 at 2 53 46 PM
> Should there be any differentiation between the two text boxes? All the comments run into each other as opposed to looking like two different entities.

Agree I think at minimum there should be some separation there, better yet maybe a gray border separating each comment.

**Review by htranho — APPROVED** [review]

**Comment by tle1989** [comment]
> Screenshot 2024-04-12 at 2 52 46 PM Screenshot 2024-04-12 at 2 53 46 PM
> Should there be any differentiation between the two text boxes? All the comments run into each other as opposed to looking like two different entities.

That is a good idea, now I have this a gray border separating each comment.
Screenshot 2024-04-15 at 3 14 48 PM

**Comment by cgokey** [comment]
I like the border, but I was thinking of more so of just a gray "bottom" border to separate each section.    I think the box is a little too much.

**Comment by tle1989** [comment]
> I like the border, but I was thinking of more so of just a gray "bottom" border to separate each section. I think the box is a little too much.
Screenshot 2024-04-16 at 10 22 01 AM

**Review by cgokey — APPROVED** [review]

**Pull request merged**